app/models/grda_warehouse/enrollment_change_history.rb

```diff
@@ -32,6 +45,7 @@ def self.attributes_for_client_on_date client:, date:
       attributes_for_client = {
         client_id: client.id,
         on: date,
+        # are timestamps and version columns used? This table is large; perhaps they could be dropped for efficiency
```

Comment:
maybe we can drop one or all of update_at, created_at, and version? Unclear if they are needed

Reply:
Hmm, they probably aren't needed, but let's see what we gain with the initial cleanup.